wgsl/index.bs

```diff
@@ -14175,7 +14251,10 @@ The number of samples per texel in the multisampled texture.
 Samples a texture.
 
 [=shader-creation error|Must=] only be used in a [=fragment=] shader stage.
-[=shader-creation error|Must=] only be invoked in [=uniform control flow=].
+
+[=shader-creation error|Must=] only be invoked in [=uniform control flow=],
+    or the call must have an [=attribute/unchecked_uniformity=] attribute,
+    or the [=option/fragmentShaderDerivativesCheckedUniformity=] option must be `false`.
```

Comment:
Editorially, it would be nice to define a term that captures these conditions, and then just use that. I think the conditions are going to get worse, not better.

Reply:
Yeah, there's a balance to be struck.
For now I left it as-is because it's fully explicit.  Adding another term for this would cause a cognitive indirection, and many folks like seeing the repetition and explicitness.
If/when these things land I'm open to editorial suggestions, starting with a name for the condition.